.github/pull_request_template.md

```diff
@@ -5,11 +5,12 @@ Please include a summary of the change and which issue is fixed. Please also inc
 
 If your pull request closes a GitHub issue, then set its number below.
 
-Fixes # (issue)
+Resolves # (issue)
 
 
 ## Checklist:
-
+* [ ] My changes are covered by tests.
+* [ ] I have run [the pre-commit hooks](https://github.com/OpenMined/PySyft/blob/master/CONTRIBUTING.md#setting-up-pre-commit-hook) to format and check my code for style issues.
```

Comment:
Q: Should we add this? The PR should fail if you did not run this.

Reply:
Yeah, I'm just tired of people who didn't run the hooks asking me why their PR fails. 😅

Reply:
Ahh ^_^. Makes sense